source/tryGoal/tryGoal_service.js

```diff
@@ -15,6 +15,50 @@ services.factory('ConfigAPI', ['$resource', function($resource) {
     });
 }]);
 
+services.factory('agentHttpInterceptor', ['agentMemory', '$rootScope', function(agentMemory, $rootScope) {
+    
+    $rootScope.apiOutput = new Array();
+
+    return {
+
+        'request': function(config) {
+            
+            if (agentMemory.tryGoal && !config.url.endsWith('.html') && 
```

Comment:
!config.url.endsWith('.html') is that needed or is it covered by the line below?

Reply:
Yes good point the lines below became necessary later but on reflection I should have removed the !html at that time. I will update.